pages/linux/vgs.md

```diff
@@ -0,0 +1,31 @@
+# vgs
+
+> Display information about volume groups.
```

Comment:
Let's also link to the man page.

Reply:
The lvmteam doesn't provide human readable man pages on their git [here](https://github.com/lvmteam/lvm2/tree/master/man). Would [this man page](https://man7.org/linux/man-pages/man8/vgs.8.html) generated from it do?

Reply:
yes.